pkg/controllers/nodeclaim/lifecycle/liveness_test.go

```diff
@@ -72,6 +72,8 @@ var _ = Describe("Liveness", func() {
 			}
```

Comment:
Do we need to add a test to ensure that we are handling reading the status condition correctly now that we have this new timeout?

Reply:
Not sure how you mean - I've added a test that tests we don't delete the nodeclaim before the timeout so we have both before and after the timeout.

The other tests still validate that if we launched successfully that we don't delete the nodeclaim for the launch timeout if we haven't registered before the registration timeout.

Reply:
What I meant was a test that validates that we are using the lastTransitionTime of the NodeClaim and not the creationTimestamp of the NodeClaim